certification/internal/policy/types.go

```diff
@@ -1,39 +0,0 @@
-package policy
-
-type Definition struct {
-	ValidatorFunc func(string) (bool, error)
-	Metadata      Metadata
-	HelpText      HelpText
-}
-
-func (pd *Definition) Validate(image string) (bool, error) {
-	return pd.ValidatorFunc(image)
-}
-
-func (pd *Definition) Meta() Metadata {
-	return pd.Metadata
-}
-
-func (pd *Definition) Help() HelpText {
-	return pd.HelpText
-}
```

Comment:
Leave this struct and its methods in place I can move it to its new location after this merges.

Reply:
I added this back. It would be great to have a more descriptive name for this struct. Not sure what "Definition" is; is it "PolicyDefinition"?

Reply:
100% agree, I'll get it renamed and relocated to be more meaningful.